selenium.go

```diff
@@ -271,6 +271,7 @@ type WebDriver interface {
 	// MaximizeWindow maximizes a window. If the name is empty, the current
 	// window will be maximized.
 	MaximizeWindow(name string) error
+	MinimizeWindow(name string) error
```

Comment:
This also needs a test, similar to the existing MaximizeWindow test:

https://github.com/tebeka/selenium/blob/51a095152aa5bbbd639eb54485dbab4308e85c57/remote_test.go#L826